scripts/build/linting.mk

```diff
@@ -79,6 +79,17 @@ gosec: gosec-install
 	@echo "--> Running gosec"
 	@gosec ./...
 
+#################
+#   vulncheck   #
+#################
+
+vulncheck-install:
+	@go install golang.org/x/vuln/cmd/govulncheck@latest
+
+vulncheck: vulncheck-install
+	@echo "--> Running govulncheck"
+	@govulncheck $(shell go list ./... | grep -v '/testing/')
```

Comment:
excluded testing files since there are a couple of vulnerabilities with Kurtosis. To be fixed